Makefile

```diff
@@ -49,7 +48,7 @@ convey: clean build
 
 .PHONY: clean
 clean:
-	rm -rf ./$(cb)/$(cb) ./$(bin)-*.zip $(test_path)
+	rm -rf ./$(cb)/$(cb) ./$(bin)-*.zip
```

Comment:
Please update the whole Makefile in line with the [styleguides](https://github.com/companieshouse/styleguides/blob/master/Makefiles/go_makefiles.md)

Reply:
The `Makefile` has been updated since merging `master` into this branch. Does this comply with standards?